app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class PostsController < ApplicationController
+  def index
+    @post = Post.new
+    @user_posts = Post.all #current_user.posts
+  end
+
+  def create
+    @user = current_user
+    @post = @user.posts.build(post_params)
+    @post.save
+    redirect_to posts_path
+  end
+
+  def show; end
+
+  def destroy
+    @post = Post.find(params[:id])
+    @post.destroy
+    redirect_to posts_path
+  end
+
+  private
+
+  def post_params
+    params.require(:post).permit(:post_text, :user_id)
```

Comment:
- [ ] **[OPTOINAL]** We don't need to permit the `user_id`. We already creating the post through association with `current_user`

Reply:
removed user_id from post_params